render.yaml

```
@@ -77,9 +74,9 @@ services:
 
 databases:
   - name: rr-postgres
-    plan: starter
+    plan: free
     region: oregon
     databaseName: recruiter_rankings_production
     user: recruiter_rankings
-    ipAllowList: []  # Only allow connections from Render services
-
+    ipAllowList: [newyork.nicklange.family]  # Only allow connections from Render services
+    plan: free
```

YAMLlint (1.37.1)

[error] duplication of key "plan" in mapping

(key-duplicates)